tests/phpunit/Performance/Time/TimeFormatterTest.php

```diff
@@ -100,23 +106,13 @@ public function minutesProvider()
         yield [122.9, '2m 2s'];
     }
 
-    /**
-     * @dataProvider minutesProvider
-     */
-    public function test_it_converts_seconds_to_hours_string(float $seconds, string $expectedString): void
-    {
-        $timeString = $this->timeFormatter->toHumanReadableString($seconds);
-
-        $this->assertSame($expectedString, $timeString);
-    }
-
-    public function hoursProvider()
+    private static function hoursProvider(): Generator
     {
         yield [3600, '1h'];
 
         yield [3600.99, '1h'];
 
-        yield [3601, '1h 0m 1s'];
```

Comment:
how does it still work? this is a breaking change but I don't see that the source code is changed

Reply:
I was as confused and its simply becasue the provider was not used...

Reply:
heh, interesting ;)